### PR TITLE
listing places to go for lunch near MSU

### PR DESCRIPTION
**Pull request by bfasy**

### CODE REVIEW

File: lunch-options.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Lunch
+
+Pickle Barrel
```
Comment (githubteacher) [inline lunch-options.md]: @bfasy, Could you turn this into a Markdown formatted bullet list? Use prefix each line with a `- `. More specifically, like this:

```
- Option 1
- Option 2
```

File: lunch-options.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Lunch
+
+- Pickle Barrel
+- Taco Montes
+- Miller
```
Comment (githubteacher) [inline lunch-options.md]: :metal: Thanks :sparkles:

Comment (CrimsonScorpion27) [inline lunch-options.md]: :8ball:

Comment (mwittie) [inline lunch-options.md]: :boom:

**Comment by mwittie** [comment]
Also add `Whistle Pig`

**Comment by sorliem** [comment]
What about Spec's?

**Comment by MacBookProne** [comment]
Don't forget Five on Black :rice_ball:

**Pull request merged**